src/forms/zendesk_config.ts

```diff
@@ -32,8 +34,8 @@ class FormFields extends BaseFormFields {
     configStore: ConfigStore
     storeValues: AppConfigStore
 
-    constructor(call: AppCall, configStore: ConfigStore, zdClient: ZDClient, mmClient: Client4) {
-        super(call, zdClient, mmClient);
+    constructor(call: AppCallRequest, configStore: ConfigStore, mmClient: Client4) {
+        super(call, {} as ZDClient, mmClient);
```

Comment:
`BaseFormFields` doesn't require a ´ZDClient`. Can we pass `undefined` here?

Reply:
yes, I had to allow `undefined` in `.eslintrc.json`.   done.